Documentation/microsoft-team.md

```diff
@@ -0,0 +1,23 @@
+# Microsoft Team Onboarding
+
+If you are new to Microsoft, you will need to onboard into various GitHub projects in order to get your work done.
+
+## Getting Permissions
+
+There are two main teams to join to get access to Microsoft .NET-related projects.
+
+* [microsoft team - dotnet org](https://repos.opensource.microsoft.com/dotnet/teams/microsoft/join/)
+* [dotnet team - microsoft org](https://repos.opensource.microsoft.com/Microsoft/teams/dotnet/join/)
+
+Once your permission requests are approved, you should have write permissions to [.NET Core Repos](https://github.com/dotnet/core/blob/master/Documentation/core-repos.md). Everyone using these repos should also be able to @mention you, too.
+
```

Comment:
Maybe add a section for tools? I suggest the following:

* [Chrome extension to see who is an FTE](https://repos.opensource.microsoft.com/settings/security/tokens/extension)
* [Chrome extension for CodeFlow on GitHub](https://chrome.google.com/webstore/detail/codeflow/aphnoipocoffpdafmiidfmaiadhilelm)